go.mod

```diff
@@ -1,17 +1,17 @@
 module go.goms.io/fleet
 
-go 1.22.2
+go 1.22.7
```

Comment:
please update other files, eg, docker file

Reply:
Docker files are already 1.22.7 https://github.com/Azure/fleet/blob/cb9a7a0b305ae8a2875fa9e3ea9fb9f8d134b27b/docker/member-agent.Dockerfile#L2

Reply:
These files were bumped by you: https://github.com/Azure/fleet/commit/a8c5a2e266dccfe14ad7cb84f7078bbf361e8a9f